Makefile

```diff
@@ -0,0 +1,36 @@
+
+FORMATTER = black
+
+.PHONY: help
+help:
+	@echo "---------------COMMANDS-----------------"
+	@echo -e "make help\nmake lint\nmake flake\nmake format\nmake bandit\nmake test"
+	@echo "------------------------------------"
+
+.PHONY: lint
```

Comment:
Please run all linting tools in the lint target. flake8, mypy, pylint, bandit, (isort and black too if they have a check-only option)

Reply:
I've added all linting tools under `make lint`